internal/clients/http/azure/types/azure_types.go

```diff
@@ -9,16 +9,15 @@ import (
 //go:embed *.yaml availability/*.yaml
 var fsTypes embed.FS
 
-var registeredTypes clients.RegisteredInstanceTypes
-var regionalAvailability clients.RegionalTypeAvailability
+var typeInfo clients.InstanceTypeInfo
```

Comment:
I moved this into a new type from the clients package so the lookup function can be shared across providers.